portfolio.css

```diff
@@ -29,10 +32,81 @@ body {
     transform: translate(-50%, -50%);
 }
 
-.page-links-container-portfolio {
```

Comment:
I have noticed this above, it seems you have more than one CSS sheet for each page, while this will work, here it created duplicate work, it is also worth checking if it is best practice.

Reply:
Charlie advised to do this. One CSS sheet for common styles across pages and one CSS sheet per individual page.